### ISSUE TITLE
Validation is skipped silently if resources.format is uppercase (CSV instead of csv)

### ISSUE THREAD

**Issue by vitorbaptista** [frictionlessdata/frictionless-py#494]
# Overview

Hey @roll, hope you're well.

I extracted a CKAN dataset as a datapackage using `ckanapi action package_show -r https://demo.ckan.org/ id=just-another-test-dataset`. This returns the following datapackage.json:

```json
{
  "author": "Test Author",
  "author_email": "test@example.com",
  "creator_user_id": "d9394fed-8637-4671-88ce-8eed35171268",
  "extras": [
    {
      "key": "Atualizado Em",
      "value": "2020/07/18"
    },
    {
      "key": "Área",
      "value": "infraestrutura"
    }
  ],
  "groups": [],
  "id": "d644520b-cdab-40cc-876d-469fc632c7ea",
  "isopen": true,
  "license_id": "cc-zero",
  "license_title": "Creative Commons CCZero",
  "license_url": "http://www.opendefinition.org/licenses/cc-zero",
  "maintainer": "Test Mantainer",
  "maintainer_email": "maintainer@example.com",
  "metadata_created": "2020-07-18T21:24:09.119107",
  "metadata_modified": "2020-07-20T12:35:59.929906",
  "name": "just-another-test-dataset",
  "notes": "Just another test dataset",
  "num_resources": 1,
  "num_tags": 1,
  "organization": null,
  "owner_org": null,
  "private": false,
  "relationships_as_object": [],
  "relationships_as_subject": [],
  "resources": [
    {
      "cache_last_updated": null,
      "cache_url": null,
      "created": "2020-07-18T21:25:21.460146",
      "datastore_active": true,
      "description": "A list of schools in Birmingham, UK, extracted from https://data.birmingham.gov.uk/dataset/birmingham-schools-datasets.",
      "format": "CSV",
      "hash": "",
      "id": "fc6bdb80-7f22-412d-b439-97e1ebc929f1",
      "last_modified": null,
      "mimetype": null,
      "mimetype_inner": null,
      "name": "Birmingham Schools",
      "package_id": "d644520b-cdab-40cc-876d-469fc632c7ea",
      "position": 0,
      "resource_type": null,
      "revision_id": "39e4c0be-64e8-44fb-b344-6d9ba71a766d",
      "size": null,
      "state": "active",
      "url": "https://raw.githubusercontent.com/vitorbaptista/birmingham_schools/master/data/birmingham_schools.csv",
      "url_type": null
    }
  ],
  "revision_id": "39e4c0be-64e8-44fb-b344-6d9ba71a766d",
  "state": "active",
  "tags": [
    {
      "display_name": "just for test",
      "id": "8e95decc-1f40-427e-9a72-2c99ce821289",
      "name": "just for test",
      "state": "active",
      "vocabulary_id": null
    }
  ],
  "title": "Just Another Test Dataset",
  "type": "dataset",
  "url": "https://example.com/test-data.csv",
  "version": "0.75"
}
```

If I save it try to validate it with `frictionless validate datapackage.json`, the CLI tool returns without output with a successful exit code. If I change the datapackage to replace `"format": "CSV"` with `"format": "csv"`, the validation runs fine.

I couldn't find on the [Data Resource spec](https://specs.frictionlessdata.io/data-resource/#metadata-properties) any mentions about the format being lowercase or uppercase. It doesn't seem to be defined.

As a user, I'd expect it to not make a difference. CSV, csv, or CsV should all work IMHO. @roll WDYT?

**Comment by roll** [comment]
Hi @vitorbaptista :smiley: Good! How are you?

Sure I'll normalize (lowercase) `resource.schema/format`. It's definitely should work no matter the case

**Comment by roll** [comment]
The update is being released as `frictionless@3.24`

BTW this data package was also really useful to realize that we had not supported deprecated `resource.url`
